src/net/sourceforge/kolmafia/request/coinmaster/SkeletonOfCrimboPastRequest.java

```diff
@@ -128,7 +130,9 @@ private static boolean applySpecial() {
     prices.entrySet().removeIf(e -> e.getKey() < ItemPool.SMOKING_POPE);
     prices.put(item.getItemId(), price);
     SKELETON_OF_CRIMBO_PAST.registerPurchaseRequests();
-
+    Concoction conc = ConcoctionPool.get(item);
+    conc.resetIngredients();
```

Comment:
this is perhaps a bit hacky

the research to realise this is the problem is good, but also the existing concoction is using an inaccessible shop (right?), so "create" shouldn't consider it available. So there's still something wrong in the code.